Net07.HW/Net07.DynamicProgrammingAndClasses/Net07.DynamicProgrammingAndClasses/Oval.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Net07.DynamicProgrammingAndClasses
+{
+    class Oval : Circle
+    {
+        public double SmallRadius { get; }
+        public Oval(Point point) : base(point)
+        {
+            SmallRadius = Radius / 2;
```

Comment:
If I create `Oval` in this way:
```
var oval = new Oval(new Point(1, 3));
```
, `Radius` value is not specified and will be 0 by default.

That's why all your constructors should accept at least 2 parameters: point and radius